x-pack/legacy/plugins/reporting/server/types.d.ts

```diff
@@ -11,16 +11,17 @@ import { PluginStart as DataPluginStart } from '../../../../../src/plugins/data/
 import { SecurityPluginSetup } from '../../../../plugins/security/server';
 import { XPackMainPlugin } from '../../xpack_main/server/xpack_main';
 import { ReportingPluginSpecOptions } from '../types';
+import { ReportingConfig, ReportingConfigType } from './core';
 
 export interface ReportingSetupDeps {
   elasticsearch: ElasticsearchServiceSetup;
   security: SecurityPluginSetup;
   usageCollection: UsageCollectionSetup;
```

Comment:
In the `kibana.json`, `usageCollection` is an `"optionalDependency"`, which is the way to go. 
`ReportingSetupDeps` should define it as an optional dependency.